GPSSetup/clients/GPSSetup/controllers/page/CameraGPS/onStart.js

```diff
@@ -0,0 +1,197 @@
+        addMarker(latlngtr, "topRightGeo");
+
+    } else {
+        // since there are no markers set them up near the camera origin
+        console.log("adding new markers");
+        latlngbl = {lat:page.data.camera.geo.coordinates[1] - 0.0001, lng:page.data.camera.geo.coordinates[0] - 0.0001};
+        addMarker(latlngbl, "bottomLeftGeo");
+
+        latlngbr = {lat:page.data.camera.geo.coordinates[1] - 0.0001, lng:page.data.camera.geo.coordinates[0] + 0.0001};
+        addMarker(latlngbr, "bottomRightGeo");
+
+        latlngtl = {lat:page.data.camera.geo.coordinates[1] + 0.0001, lng:page.data.camera.geo.coordinates[0] - 0.0001};
+        addMarker(latlngtl, "topLeftGeo");
+
+        latlngtr = {lat:page.data.camera.geo.coordinates[1] + 0.0001, lng:page.data.camera.geo.coordinates[0] + 0.0001};
+        addMarker(latlngtr, "topRightGeo");
+
+    }
+
+    // add markers function
+    function addMarker(location, title) {
+        var marker = new google.maps.Marker({
+            position: location,
+            title: title,
+            map: map,
+            draggable:true
+        });
+
+
+        marker.addListener('dragend', function(){
+            console.log(this);
+
+            var location = this.position; 
+
+            // set the geoJSON for the object
+            page.data.camera.gpsTrack[this.title] = {
+                "lng": location.lng(),
+                "lat": location.lat()
+            };
+
+        }); 
+
+        page.data.markers.push(marker);
+        //marker.setMap(map);
+    }
+
+    function moveShapes(){      
+        if(page.data.camera.imageTrack.topLeftImage){            
+            var x = page.data.camera.imageTrack.topLeftImage.x;
+            var y = page.data.camera.imageTrack.topLeftImage.y;
+            page.data.CanvasState.moveShape(0, x,y);
+        } else {
+
+            page.data.CanvasState.moveShape(0, 20,20);
+        }
+
+        if(page.data.camera.imageTrack.bottomLeftImage){
+            var x = page.data.camera.imageTrack.bottomLeftImage.x;
+            var y = page.data.camera.imageTrack.bottomLeftImage.y;
+            page.data.CanvasState.moveShape(1, x,y);
+
+        } else {
+
+            page.data.CanvasState.moveShape(1, 20,80);
+        }
+
+        if(page.data.camera.imageTrack.bottomRightImage){
+            var x = page.data.camera.imageTrack.bottomRightImage.x;
+            var y = page.data.camera.imageTrack.bottomRightImage.y;
+            page.data.CanvasState.moveShape(2, x,y);
+
+        } else {
+
+            page.data.CanvasState.moveShape(2, 80,80);
+
+        }    
+
+        if(page.data.camera.imageTrack.topRightImage){
+            var x = page.data.camera.imageTrack.topRightImage.x;
+            var y = page.data.camera.imageTrack.topRightImage.y;
+            page.data.CanvasState.moveShape(3, x,y);
+
+        } else {
+            page.data.CanvasState.moveShape(3, 80,20);
+        }
+
+
+    }
```

Comment:
Remove unnecessary new lines between if/else, and inside else blocks. Also change if statements to the following format: 

```if (myCondition) {``` 

(space after `if` and space before open bracket)